src/pages/LoadingPage.tsx

```diff
@@ -58,7 +58,11 @@ const LoadingPage: React.FC<LoadingPageProps> = ({ onLoad, logType }) => {
   const { data, error } = useAxiosGet(url);
   useEffect(() => {
     if (data) {
-      ingestLines(data.split("\n"), logType);
+      // Temporary fix
+      ingestLines(
+        data.split("\n").filter((l) => l !== ""),
+        logType
+      );
```

Comment:
I'm not sure if this is just an error with the log files from S3, but splitting on the new line causes the last line to be a whitespace which doesn't render in the virtualized list

Reply:
This will remove intentional blank lines.

Most files have new line characters at the end. https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file